src/app/component/dashboard/dashboard.component.ts

```diff
@@ -36,19 +30,10 @@ export class DashboardComponent implements OnInit,OnDestroy {
     public questionService: QuestionService,
   ){}
   
-  mapJSONData(){
-    // this.title = this.quizData?.QuizType?.title;
-    // this.image = this.quizData?.QuizType;
-    // this.description = this.quizData?.QuizType;
-    // console.log("title",this.title)
-    // console.log("image",this.image)
-    // console.log("description",this.description)
+  formatData() {
```

Comment:
Could you please remove this function as there is nothing in this

Reply:
yes i removed it